01.fizzbuzz/fizzbuzz.rb

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env ruby
+(1..20).each do |x|
+  if (x%3 == 0 && x%5 != 0)
+    puts "Fizz"
+  elsif (x%5 == 0 && x%3 != 0)
+    puts "Buzz"
+  elsif (x%3 == 0 && x%5 == 0)
```

Comment:
この条件を一番上に書けば、他の条件をシンプルにできます。つまり
```suggestion
  if x % 3 == 0 && x % 5 == 0 
    ...
  elsif x % 3
    ...
  elsif x % 5
```
とかけますね。どちらでも動作は変わりませんが、こちらの方が良いコードと言えると思います。参考までに。